src/components/Header/Header.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react'
+import {
+  Header as USWDSHeader,
+  Title,
+  PrimaryNav,
+  NavMenuButton,
+} from '@trussworks/react-uswds'
+
+import styles from './Header.module.scss'
+
+import Logo from 'components/Logo/Logo'
+import NavLink from 'components/util/NavLink/NavLink'
+import LinkTo from 'components/util/LinkTo/LinkTo'
+
+const Header = () => {
+  const [expanded, setExpanded] = useState(false)
+  const handleNavButtonClick = (): void =>
+    setExpanded((prevExpanded) => !prevExpanded)
+
+  const navItems = [
+    <NavLink href="#" key="nav_about">
+      <span>About Space Force</span>
```

Comment:
Question for my own code: Is wrapping the text inside of a NavLink in a span best practice? Or is it for additional styling? Would that go for <LinkTo> or other similar link components?

Reply:
OH this is purely because this is how the USWDS Header styles top-level nav links (see the [source code](https://designsystem.digital.gov/components/header/)). that is literally the only reason, so no need to worry about it in other contexts.